internal/request/validator.go

```diff
@@ -80,15 +93,9 @@ func (v *Validator) ParseTTL(ctx context.Context, ttl string) (context.Context,
 }
 
 // Validate checks that the request inside this validator is valid and returns
-// either the validated request or an error.
+// either the validated request or an error. This should be used to check all
+// requests, prior to the RPC type specifc validation.
```

Comment:
I would like it if we could rename this and the other Validate function to
something else to make it obvious that this is only part of the validation but
I can't think of an obvious name here. If you can think of one, go for it.
Otherwise leave it as is.